src/event/mod.rs

```diff
@@ -388,7 +388,7 @@ impl From<Bytes> for Event {
             .insert(log_schema().message_key(), message);
         event
             .as_mut_log()
-            .insert(log_schema().timestamp_key(), Utc::now());
+            .insert(Atom::from(log_schema().timestamp_key()), Utc::now());
```

Comment:
Why is this `Atom::from` needed here and other places we are using `log_schema()` fields with `.insert`?

Reply:
The goal is to remove `DefaultAtom`/`Atom` completely, but doing it in one chunk would be a very large change, this preps the change a bit. :)

Reply:
:+1: why would it be needed for `timestamp_key()` but not `message_key()` (appearing above).